server/src/test/java/org/opensearch/gateway/remote/RemoteClusterStateServiceTests.java

```diff
@@ -265,7 +274,46 @@ public void testWriteFullMetadataSuccess() throws IOException {
         assertFalse(manifest.getCustomMetadataMap().isEmpty());
```

Comment:
can we add assert here that ephemeral objects are empty?

Reply:
Added